webapp/src/js/services/resource-icons.js

```diff
@@ -95,6 +95,8 @@ angular.module('inboxServices').factory('ResourceIcons',
       callback: change => updateResources(change.id)
     });
 
+    var init = updateResources(DOC_IDS[0]);
```

Comment:
Because of this line, the `resources` doc will always be downloaded twice, as it's already downloaded along with the other docs on line 90:

```
DOC_IDS.forEach(doc => updateResources(doc));
```